core/src/main/resources/default-main.conf

```diff
@@ -13,8 +14,10 @@
 # remote.nossl.hosts = *.redhat.com, 10.0.*
 ######################################################################
 
-# remote.nossl.hosts = "10.*,172.*,localhost"
+# Regexes of allowed NON-SSL remote hosts separated by ","
+remote.nossl.hosts = localhost,*.apache.org,127.0.0.1
```

Comment:
Hmm, not sure if we should give apache.org a pass on this.

Reply:
i have put *.apache.org because of our main default maven repo - central which is entered as "repo.maven.apache.org"

Reply:
Yep, but I think we'll want to force SSL use for that too. We should update the default repos that Indy creates too.